app/src/main/java/space/taran/arkmemo/ui/adapters/NotesListAdapter.kt

```diff
@@ -29,13 +32,13 @@ class TextNotesListAdapter(private val notes: List<TextNote>): RecyclerView.Adap
     }
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): NoteViewHolder {
-        val itemView = LayoutInflater.from(parent.context).inflate(R.layout.text_note, parent, false)
+        val itemView = LayoutInflater.from(parent.context).inflate(R.layout.note, parent, false)
         return NoteViewHolder(itemView)
     }
 
     override fun onBindViewHolder(holder: NoteViewHolder, position: Int) {
-        holder.title.text = notes[position].content.title
-        holder.date.text = notes[position].meta?.modified?.toString() ?: "Just now"
+        holder.title.text = notes[position].resourceContent.title
```

Comment:
I haven't looked deeply yet, but why `title` is part of `content`? Title and description should be stored as properties.

Reply:
You are right, we need properties field in note. I will add it